packages/IMAPClient-UI.package/ICFolderDialog.class/instance/login..st

```diff
@@ -0,0 +1,10 @@
+action - emails
+login: anICEndpoint
+
+	| continueAsking | 
+	continueAsking := false. 
+	[continueAsking] whileFalse: [
```

Comment:
If you call the variable `continueAsking`, I would have expected a `whileTrue` loop. Maybe rename that variable so that the code is more intuitive.

Reply:
thank you for the comment, I almost did not get it. It should be addressed.